### PR TITLE
Jittering etc.

### PR DESCRIPTION
**Pull request by djmannion**
Adds jittering to the fits, with updates to the article and the addition of a changelog.

### CODE REVIEW

**Comment by djmannion** [comment]
Had to upgrade my R dependencies locally to get the plot tests to match.

File: R/fit_data.R [hunk 1]
```diff
@@ -16,13 +16,22 @@
 #'  fit by seeking its minimum.
 #'   * `ks`: Kolmogorov-Smirnov statistic.
 #'   * `likelihood`: Negative log-likelihood.
+#' @param jitter_settings Settings for running the fitting multiple times with
+#'   randomly-generated offsets ('jitter') applied to the starting estimates.
+#'   * `n`: How many jitter iterations to run (default of 7); the total number
+#'   of fits is `n + 1` (because the un-jittered start points are also fit).
+#'   * `prop`: The maximum jitter offset, as a proportion of the start
+#'   value (default of 0.5).
+#'   * `seed`: Seed for the random jitter generator (default is unseeded).
```
Comment (mariadelmarq) [inline R/fit_data.R:25]: Question: what does it mean for the default to be unseeded? Is it that it chooses a random seed, that is not saved or displayed in any way, or something else?

File: R/fit_data.R [hunk 2]
```diff
@@ -525,3 +629,89 @@ add_ecdf_to_data <- function(data) {
...
+    jitter_amount_prop,
+    n_jitters,
+    seed = NA) {
+  if (n_jitters == 0) {
+    return(list())
+  }
+
+  seed <- ifelse(
+    is.na(seed),
+    sample.int(n = .Machine$integer.max, size = 1),
```
Comment (mariadelmarq) [inline R/fit_data.R:646]: Here's my answer to the above!

File: R/fit_data.R (hunk 1 above)
Comment (mariadelmarq) [reply R/fit_data.R:25]: Never mind, I answered it below :)

**Review by mariadelmarq — APPROVED** [review]
Brilliant, thanks heaps!

**Comment by mariadelmarq** [comment]
@djmannion Should we be incrementing the minor to 0.2.0 for this release?

**Comment by djmannion** [comment]
> @djmannion Should we be incrementing the minor to 0.2.0 for this release?

I wasn't sure if there was anything else you wanted to add for a 0.2.0 release so kept this branch at 'development'.

**Comment by djmannion** [comment]
But I think it certainly warrants bumping up to 0.2.0 irrespective of any other changes.

**Pull request merged**